### PR TITLE
Add docs for pandas API.

### PR DESCRIPTION
**Pull request by tjann**

### CODE REVIEW

**Comment by tjann** [comment]
This is how it is structured

image

- pandas is not nested in python
- pandas does not repeat docs from python
- the explanation of their relation is here: https://github.com/datacommonsorg/docsite/pull/55/files#diff-80ff7d1a7969be4745beec1dbe6347e3R10-R14
  - it is short, I am not sure if this is enough, hoping to get opinion from folks who did not write the pandas pkg--does that make sense/ is it clear how to use python functions in pandas?

File: api/pandas/covariate_dataframe.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+---
+layout: default
+title: Covariates for Places as pd.DataFrame
```
Comment (shifucun) [inline api/pandas/covariate_dataframe.md:3]: should this be covariates between place and stat var?

File: api/pandas/covariate_dataframe.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+A `pandas.DataFrame` with [`places`](https://datacommons.org/browser/Place)
+(str)
+as index and [`stat_vars`](https://datacommons.org/browser/StatisticalVariable)
+(str) as columns, where each cell is latest observed statistic (float) for
+its `Place` and `StatisticalVariable`.
+
+**Raises**
+
+* `ValueError` - If no statistial values found for with the given parameters.
```
Comment (shifucun) [inline api/pandas/covariate_dataframe.md:39]: remove "with"

File: api/pandas/time_series.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+nav_order: 1
+parent: Pandas
+grand_parent: API
+---
+
+# Get Time Series as pandas Series
+
+## `datacommons_pandas.build_time_series(place, stat_var, measurement_method=None,observation_period=None, unit=None, scaling_factor=None)`
+
+Returns a `pandas.Series` mapping date to value for a `place` based on the
```
Comment (shifucun) [inline api/pandas/time_series.md:13]: not sure if "mapping" is used in pandas or somether Series specific words

Comment (tjann) [reply api/pandas/time_series.md:13]: Changed this block to "
Returns a `pandas.Series` representing a time series for the [`place']https://datacommons.org/browser/Place] and
[`stat_var`](https://datacommons.org/browser/StatisticalVariable) satisfying any optional parameters.
"

File: api/pandas/covariate_dataframe.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+layout: default
+title: Covariates for Places as pd.DataFrame
+nav_order: 3
+parent: Pandas
+grand_parent: API
+---
+
+# Get Covariates for Places as pandas DataFrame
+
+## `datacommons_pandas.build_covariate_dataframe(dcids, stats_vars)`
```
Comment (beets) [inline api/pandas/covariate_dataframe.md:11]: nit: we used places instead of dcids in the REST api

File: api/pandas/covariate_dataframe.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+**Raises**
+
+* `ValueError` - If no statistial values found for the given parameters.
+
+Be sure to initialize the library. See the
+[datacommons_pandas library setup guide](/api/pandas/) for more details.
+
+You can find a list of `StatisticalVariable`s with human-readable names [here](/statistical_variables.html).
+
+## Examples
```
Comment (beets) [inline api/pandas/covariate_dataframe.md:46]: please add an example for an overly constrained query that results in a value error

File: api/pandas/index.md [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+title: Pandas
+nav_order: 3
+parent: API
+has_children: true
+---
+# Data Commons Pandas API
+
+The **Data Commons Pandas API** is a superset of the Data Commons Python API:
+all functions from the Python API are also accessible from
+the Pandas API, and supplemental functions help with directly creating pandas
```
Comment (beets) [inline api/pandas/index.md:12]: nit: add link for "pandas", and should it be Pandas?

File: api/pandas/time_series.md [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+nav_order: 1
+parent: Pandas
+grand_parent: API
+---
+
+# Get Time Series as pandas Series
+
+## `datacommons_pandas.build_time_series(place, stat_var, measurement_method=None,observation_period=None, unit=None, scaling_factor=None)`
+
+Returns a `pandas.Series` representing a time series for the [`place']https://datacommons.org/browser/Place] and
```
Comment (beets) [inline api/pandas/time_series.md:13]: nit: wrong closing tick for place

File: api/pandas/covariate_dataframe.md (hunk 4 above)
Comment (tjann) [reply api/pandas/covariate_dataframe.md]: Thanks for the catch

File: api/pandas/covariate_dataframe.md (hunk 5 above)
Comment (tjann) [reply api/pandas/covariate_dataframe.md]: There are no optional constraints for this function

File: api/pandas/index.md (hunk 6 above)
Comment (tjann) [reply api/pandas/index.md:12]: pandas is technically lower case, but I've upper cased it for the "Pandas API".  I added a link.

File: api/pandas/time_series.md (hunk 7 above)
Comment (tjann) [reply api/pandas/time_series.md]: Thanks, fixed

File: api/pandas/covariate_dataframe.md (hunk 5 above)
Comment (tjann) [reply api/pandas/covariate_dataframe.md]: I will add one with no data

Comment (tjann) [reply api/pandas/covariate_dataframe.md]: I added an example for no data for both time series and multivariate data frame.

**Pull request merged**